awx/ui_next/src/screens/Inventory/shared/InventorySourceSubForms/SCMSubForm.jsx

```diff
@@ -126,6 +126,7 @@ const SCMSubForm = ({ autoPopulateProject, i18n }) => {
           }}
           aria-label={i18n._(t`Select source path`)}
           placeholder={i18n._(t`Select source path`)}
+          createText={i18n._(t`Set source path to`)}
```

Comment:
I noticed the text shown when manually setting an inventory source is kind of weird:

![Screenshot from 2021-03-25 18-32-23](https://user-images.githubusercontent.com/9753817/112552165-b9ff6500-8d98-11eb-8966-7b0d2086aa88.png)

"Create foo/bar" almost makes it seem like a path would be created somewhere. When we type in a path like this, we're really just specifying a file location.

Our component library has a [parameter for customizing the text](https://www.patternfly.org/2020.04/documentation/react/components/select] so I changed it:

![Screenshot from 2021-03-25 18-30-54](https://user-images.githubusercontent.com/9753817/112552378-12366700-8d99-11eb-906b-49f0f013b48e.png)

Reply:
I like this phrasing